codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -365,13 +361,23 @@ void window(int initialWindowSize) {
          */
         void allocate(int bytes) {
             allocated += bytes;
+            // Also artificially reduce the streamable bytes for this tree to give the appearance
+            // that the data has been written. This will be restored when we write the allocated bytes.
```

Comment:
`This will be restored when we write the allocated bytes.` - > `This will be restored before the allocated bytes are actually written.`?